spec/components/theme_store/tgz_exporter_spec.rb

```diff
@@ -0,0 +1,106 @@
+require 'rails_helper'
+require 'theme_store/tgz_exporter'
+
+describe ThemeStore::TgzExporter do
+  let(:theme) do
+    Fabricate(:theme, name: "Header Icons").tap do |theme|
+      theme.set_field(target: :common, name: :body_tag, value: "<b>testtheme1</b>")
+      theme.set_field(target: :settings, name: :yaml, value: "somesetting: test")
+      theme.set_field(target: :mobile, name: :scss, value: 'body {background-color: $background_color; font-size: $font-size}')
+      theme.set_field(target: :translations, name: :en, value: { en: { key: "value" } }.deep_stringify_keys.to_yaml)
+      image = file_from_fixtures("logo.png")
+      upload = UploadCreator.new(image, "logo.png").create_for(-1)
+      theme.set_field(target: :common, name: :logo, upload_id: upload.id, type: :theme_upload_var)
+      theme.build_remote_theme(remote_url: "", about_url: "abouturl", license_url: "licenseurl")
+
+      cs1 = Fabricate(:color_scheme, name: 'Orphan Color Scheme', color_scheme_colors: [
+        Fabricate(:color_scheme_color, name: 'header_primary',  hex: 'F0F0F0'),
+        Fabricate(:color_scheme_color, name: 'header_background', hex: '1E1E1E'),
+        Fabricate(:color_scheme_color, name: 'tertiary', hex: '858585')
+      ])
+
+      cs2 = Fabricate(:color_scheme, name: 'Theme Color Scheme', color_scheme_colors: [
+        Fabricate(:color_scheme_color, name: 'header_primary',  hex: 'F0F0F0'),
+        Fabricate(:color_scheme_color, name: 'header_background', hex: '1E1E1E'),
+        Fabricate(:color_scheme_color, name: 'tertiary', hex: '858585')
+      ])
+
+      theme.color_scheme = cs1
+      cs2.update(theme_id: theme.id)
+
+      theme.save!
+    end
+  end
+
+  let(:dir) do
+    tmpdir = Dir.tmpdir
+    dir = "#{tmpdir}/#{SecureRandom.hex}"
+    `mkdir #{dir}`
```

Comment:
Any reason you aren't using `FileUtils` for creating and removing the directories? I know these are tests but it still seems safer to me.

Reply:
No reason, other than copy/paste from some existing theme import specs. I've switched to using `FileUtils` 👍